src/main/scala/is/hail/sparkextras/ContextRDD.scala

```diff
@@ -169,6 +169,8 @@ class ContextRDD[C <: AutoCloseable, T: ClassTag](
 
   def getNumPartitions: Int = rdd.getNumPartitions
 
+  def partitions: Array[Partition] = rdd.partitions
```

Comment:
This is currently unused.  I'm not sure how I see how it is going to be used (outside getting the number of partitions).  Can you elucidate?

Reply:
I imagine the partitions was added and then removed in the process of development. I've removed it from this PR. If it's needed later, I'll reconsider.

Reply:
Oh, I see, I added it because `RVD` had `def partitions`. I also removed all uses of `RVD.partitions` (replacing them with `getNumPartitions`)